app/controllers/Shipping.scala

```diff
@@ -23,54 +22,21 @@ object Shipping extends Controller {
     )
 
   def viewCollectionPaperDigital() = CachedAction {
-    val plans = List.empty //catalog.voucher.productPlans.filter(_.products.others.contains(Digipack)).map(planToOptions)
     index(CollectionSubscriptionProduct(
       title = "Paper + digital voucher subscription",
       description = "Save money on your newspapers and digital content. Plus start using the daily edition and premium live news app immediately.",
       packageType = "paper-digital",
-      options = plans.nonEmpty.option(plans).getOrElse(
-        Seq(
-          SubscriptionOption("everyday",
-            "Everyday+", 11.99f, Some("36%"), 51.96f, "Guardian and Observer papers, plus tablet editions and Premium mobile access",
```

Comment:
The saving % values are on the corresponding Zuora rate plans in the Saving__c custom field.